CHANGELOG

```diff
@@ -2,6 +2,13 @@
 Changelog
 *********
 
+0.45.0 (unreleased)
```

Comment:
@sloria, sorry about that. This is resolved, and build should be passing before lunch.